pom.xml

```
@@ -645,7 +645,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>github-api</artifactId>
-            <version>1.90</version>
+            <version>1.111-SNAPSHOT</version>
```

When it is released...
```suggestion
            <version>1.111</version>
```